### PR TITLE
refs #18146, change default dojox/mobile theme in 1.11 to ios7 theme

### PR DESCRIPTION
**Pull request by dylans**

### CODE REVIEW

File: mobile/deviceTheme.js [hunk 1]
```diff
@@ -75,18 +75,18 @@
 		//
 		//		In this case, the following files will be loaded.
 		//
-		//	|	dojox/mobile/themes/iphone/base.css
+		//	|	dojox/mobile/themes/ios7/base.css
 		//	|	com/acme/themes/iphone/MyWidget.css
```
Comment (AdrianVasiliu) [inline mobile/deviceTheme.js]: `iphone` => `ios7`  here too (as at line 89), isn't it?
(No need to update the PR, I can add it when merging.)

Comment (dylans) [inline mobile/deviceTheme.js]: Thanks, yes, I missed this one.

Comment (AdrianVasiliu) [inline mobile/deviceTheme.js]: ok, no problem.

**Comment by AdrianVasiliu** [comment]
Merged into master via 676462997e50ea2be321ad4ee98f533635277561.

Livedocs would also need to be updated accordingly, so I pushed 8758c106af4d184e8b302021618e5aaab53ea478.

**Pull request closed without merging**